lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb

```diff
@@ -836,6 +879,10 @@ def next_scalar_value_for(scope, previous_value)
             available_values.keys.last
           elsif polymorphic_type_attribute?(scope, previous_value)
             Uniqueness::TestModels.create(previous_value).to_s
+          elsif @options[:alternatives] && scope == @attribute
+            alternatives = @options[:alternatives]
```

Comment:
We could simplify the alternatives assignment using the `Array#wrap` method.
```suggestion
            alternatives = Array.wrap(@options[:alternatives])
```